frameworks/rooch-framework/sources/transaction_validator.move

```diff
@@ -128,10 +128,10 @@ module rooch_framework::transaction_validator {
         };
         let bitcoin_addr = auth_validator::get_bitcoin_address_from_ctx();
         address_mapping::bind_bitcoin_address(sender, bitcoin_addr); 
-        let tx_sequence_info = tx_context::get_attribute<TransactionSequenceInfo>();
+        let tx_sequence_info = tx_context::get_attribute<TransactionSequenceInfoV2>();
```

Comment:
We must add TransactionSequenceInfo and TransactionSequenceInfoV2 both to the tx_context because the rust code is upgraded first, then the framework is upgraded. The old framework will read the  TransactionSequenceInfo.

https://github.com/rooch-network/rooch/blob/885a00dec2361d11ecc58af55ef25a4377fd8371/crates/rooch-pipeline-processor/src/actor/processor.rs#L152-L154

Reply:
所以升级流程是 这样？
1，rust 里也需要保留TransactionSequenceInfo和TransactionSequenceInfoV2，然后同时将TransactionSequenceInfo和TransactionSequenceInfoV2添加到 tx_context；
2，升级framework前，framework 读取TransactionSequenceInfo；
3，升级 framework 后，framework 读取TransactionSequenceInfoV2；

Reply:
对。

Reply:
有个问题，rust 里TransactionSequenceInfo和TransactionSequenceInfoV2共存的话，rust里处理交易使用哪个，TransactionSequenceInfo or TransactionSequenceInfoV2？

Reply:
rust 里应该用 TransactionSequenceInfoV2